### PR TITLE
Add Azure Login Action Step

### PR DESCRIPTION
**Pull request by simone-bennett**
When following this article we took some time to figure out that there was a step missing. Added the azure login action in.

### CODE REVIEW

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Review by M2crash1090 — APPROVED** [review]

**Comment by KarlErickson** [comment]
@juliakm ping

**Comment by juliakm** [comment]
Thanks @simone-bennett for the helpful addition! 

#sign-off

File: articles/github/github-key-vault.md [hunk 1]
```diff
@@ -94,6 +94,9 @@ Secrets fetched are set as outputs and also as environment variables. Variables
         secrets: 'mySecret'  # comma separated list of secret keys to fetch from key vault 
       id: myGetSecretAction # ID for secrets that you will reference
 ```
+## Add the Azure Login Action
+
+For GitHub actions that don't use public endpoints, you may need to configure the [Azure Login Action.](https://docs.microsoft.com/en-us/azure/developer/github/connect-from-azure?tabs=azure-cli%2Cwindows#use-the-azure-login-action)
```
Comment (KarlErickson) [inline articles/github/github-key-vault.md:99]:
```suggestion
For GitHub actions that don't use public endpoints, you may need to configure the [Azure Login Action.](connect-from-azure.md?tabs=azure-cli%2Cwindows#use-the-azure-login-action)
```

**Pull request merged**